transports/index.md

```diff
@@ -1,72 +1,38 @@
 ---
 title: Transports
 summary: An overview of the transports available in NServiceBus
-reviewed: 2018-08-26
+reviewed: 2018-11-27
 redirects:
  - nservicebus/nservicebus-and-websphere-sonic
  - nservicebus/transports
 related:
  - samples/custom-transport
 ---
 
-NServiceBus is built on top of existing queuing technologies. In NServiceBus, the choice of queuing technology is referred to as a "Transport".
+NServiceBus contains an abstraction for underlying queueing technologies. An implementation of that abstraction for a given queueing technology is known as a "transport".
 
+The transport abstraction enables business to build systems with the Particular Service Platform using existing queueing technologies. Sometimes the choice of queueing technology may be constrained be constrained by the business. At other times the best queueing technology can be selected according to requirements. The transport abstraction allows more time to be spent delivering features and less time to be spent dealing with the details of a specific queueing technology.
 
-## Types of transports
+The documentation related to transports describes the differences between them, and contains guidelines for deciding which transport may best suit a specific scenario. For each transport, the documentation describes how to make optimal usage of NServiceBus in combination with that transport. This includes how to deal with concurrency, transactions, scaling, and more.
```

Comment:
```suggestion
The transports documentation describes differences between transports, and contains guidelines for deciding which transport may best suit a specific scenario. For each transport, documentation describes how to make optimal usage of NServiceBus in combination with that transport. This includes how to deal with concurrency, transactions, scaling, and more.
```

> For each transport, documentation describes how to make optimal usage of NServiceBus in combination with that transport. 

Is this really true or this documentation is about selecting the transport based on users deployment constraints?

Reply:
Changed to

> The transport documentation describes differences between transports, and contains guidelines for deciding which transport may best suit a specific scenario. For each transport, the documentation describes how to make optimal usage of NServiceBus in combination with that transport. This includes how to deal with concurrency, transactions, scaling, and more.

Reply:
> Is this really true or this documentation is about selecting the transport based on users deployment constraints?

The new guidance material is about that, but we still have this within the documentation _for each transport_.